### PR TITLE
Switch references of /var/run -> /run

### PR DESCRIPTION
**Pull request by rhatdan**
Systemd is now complaining or mentioning /var/run as a legacy directory.
It has been many years where /var/run is a symlink to /run on all
most distributions, make the change to the default.

Partial fix for https://github.com/containers/podman/issues/8369

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Review by vrothberg — COMMENTED** [review]
LGTM

File: test/system/400-unprivileged-access.bats [hunk 1]
```diff
@@ -75,7 +75,7 @@ EOF
     is "$output" "/var/lib/containers/storage" "GraphRoot in expected place"
     GRAPH_ROOT="$output"
     run_podman info --format '{{.Store.RunRoot}}'
-    is "$output" "/var/run/containers/storage" "RunRoot in expected place"
+    is "$output" "/run/containers/storage" "RunRoot in expected place"
```
Comment (edsantiago) [inline test/system/400-unprivileged-access.bats:78]: I think you want to stet this. From what I'm seeing, `RunRoot` goes a lot deeper and will require a lot more changes.

**Comment by mheon** [comment]
LGTM

File: test/system/400-unprivileged-access.bats (hunk 1 above)
Comment (rhatdan) [reply test/system/400-unprivileged-access.bats]: I changed the match to match .*/run/containers/storage

File: docs/source/markdown/podman-build.1.md [hunk 2]
```diff
@@ -305,6 +305,11 @@ Recognized formats include *oci* (OCI image-spec v1.0, the default) and
 Note: You can also override the default format by setting the BUILDAH\_FORMAT
 environment variable.  `export BUILDAH_FORMAT=docker`
 
+#### **--from**
+
+Overrides the first `FROM` instruction within the Containerfile.  If there are multiple
+FROM instructions in a Containerfile, only the first is changed.
+
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-build.1.md:312]: Looks like we need to get a Buildah release together soon and vendored....

**Comment by TomSweeneyRedHat** [comment]
Changes LGTM, but all kinds of test unhappiness.

**Pull request merged**